jekyll/_cci2/api-developers-guide.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: classic-docs
+title: "CircleCI API Developer's Guide"
+short-title: "Developer's Guide"
+description: "API cookbook for internal and external CircleCI developers"
+categories: [getting-started]
+order: 1
+---
+
+This *API Developer's Guide* was written to assist developers in quickly and easily making API calls to CircleCI services to return detailed information about users, pipelines, projects and workflows.
+
+* TOC
+{:toc}
+
+# API Overview
+
+The CircleCI platform provides a powerful API that enables users to retrieve detailed information about users, jobs, workflows and pipelines.
+
+CircleCI API v2 enables you to use endpoints with several new features that improve the API experience, in addition to optimizing how you use the API for your jobs. Please note that CircleCI API v2 is currently in active development, therefore, the stability of the API is referred to as “mixed”.
+
+## API Classes
```

Comment:
I suggest you don't try to keep this in sync -- let the generated docs speak for themselves on this.

Reply:
Removed from doc.